pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -445,6 +445,10 @@ func CollectVmisVnicInfo(vmi *k6tv1.VirtualMachineInstance) []operatormetrics.Co
 	networks := vmi.Spec.Networks
 
 	for _, iface := range interfaces {
+		model := "<none>"
+		if iface.Model != "" {
```

Comment:
I dont think this is valid, the interface Model cannot be empty, it defaults to "virtio".
https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/converter/network.go#L55
https://github.com/kubevirt/kubevirt/blob/main/pkg/network/admitter/netiface_test.go#L135-L157

@EdDev @orelmisan PTAL

Reply:
@ormergi, per my understanding from the PR description, the metric reports the model that was specified by the user (and not what is actually used under the hood).
@avlitman could you please clarify?

Reply:
Thanks for explaining
 
@avlitman yes if default change metric should also change. Although its not likely to change IMO, and in general changing default is not trivial change and may break backward compatibility.

When model is empty in the VM/VMI object Kubveirt will set virtio model in the domain XML (the guest).